src/GlobalErrorHandler.js

```diff
@@ -21,26 +23,65 @@ class GlobalErrorHandler extends React.Component {
     return (
       <Main>
         <In>
-          <ErrorCard
-            title="Oops."
-            detailsTitle={error.message}
-            detailsContent={errorStack}
-            supportUrl="https://github.com/aragon/aragon/issues/new"
-            showReloadButton
-          >
-            Something went wrong and the application crashed. Reloading might
-            solve the problem, or you can{' '}
-            <SafeLink href="https://github.com/aragon/aragon/issues/new">
-              create an issue
-            </SafeLink>{' '}
-            on GitHub so we can help.
-          </ErrorCard>
+          {error.name === 'DAONotFound' ? (
+            <DAONotFound dao={error.dao} />
+          ) : (
+            <GenericError
+              detailsTitle={error.message}
+              detailsContent={errorStack}
+            />
+          )}
         </In>
       </Main>
     )
   }
 }
 
+const CURRENT_DOMAIN = 'app.aragon.org'
+const OLD_DAO_DOMAIN = 'app-v05.aragon.org'
+const DEPRECATION_URL =
+  'https://blog.aragon.org/deprecation-notice-on-v0-5-rinkeby-daos'
+
+const DAONotFound = ({ dao }) => (
```

Comment:
What do you think about putting `DAONotFound` and `GenericError` into somewhere else, like `components/Error/`?